packages/docusaurus-theme-classic/codeTranslations/he.json

```diff
@@ -0,0 +1,55 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "לסגור",
+  "theme.CodeBlock.copied": "הועתק",
+  "theme.CodeBlock.copy": "להעתיק",
+  "theme.CodeBlock.copyButtonAriaLabel": "להעתיק קוד",
+  "theme.NotFound.p1": "אנחנו לא מוצאים מה שאתה מנסה לחפש.",
+  "theme.NotFound.p2": "נא פנה למנהל האתר מאיפה קיבלת קישור הזה ותגיד לו שקישור לא תקין.",
+  "theme.NotFound.title": "דף לא נמצא",
+  "theme.Playground.liveEditor": "Live Editor",
+  "theme.Playground.result": "תוצאה",
+  "theme.PwaReloadPopup.closeButtonAriaLabel": "לסגור",
+  "theme.PwaReloadPopup.info": "קיימת גרסא חדשה",
+  "theme.PwaReloadPopup.refreshButtonText": "לרענן",
+  "theme.SearchBar.label": "חיפוש",
+  "theme.SearchPage.algoliaLabel": "חיפוש by Algolia",
+  "theme.SearchPage.documentsFound.plurals": "נמצא מסמך אחד|{count} מסמכים נמצאו",
+  "theme.SearchPage.emptyResultsTitle": "חפש בדוקומנטאציה",
+  "theme.SearchPage.existingResultsTitle": "תוצאות חיפוש עבור \"{query}\"",
+  "theme.SearchPage.fetchingNewResults": "מקבל תוצאות חדשות...",
+  "theme.SearchPage.inputLabel": "חיפוש",
+  "theme.SearchPage.inputPlaceholder": "הקלד כאן לחיפוש",
+  "theme.SearchPage.noResultsText": "לא נמצאו תוצאות",
+  "theme.blog.paginator.navAriaLabel": "רשימת דפים של הבלוג",
+  "theme.blog.paginator.newerEntries": "הכי חדש",
+  "theme.blog.paginator.olderEntries": "ישן יותר",
+  "theme.blog.post.paginator.navAriaLabel": "רשימת חדשות של הבלוג",
+  "theme.blog.post.paginator.newerPost": "הכי חדש",
+  "theme.blog.post.paginator.olderPost": "ישן יותר",
+  "theme.blog.post.plurals": "חדשה אחד|{count} חדשות",
+  "theme.blog.post.readMore": "עוד...",
+  "theme.blog.post.readingTime.plurals": "דקה לקריאה|{readingTime} דקות לקריאה",
+  "theme.blog.tagTitle": "{nPosts} עם תג \"{tagName}\"",
+  "theme.common.editThisPage": "לערוך דף",
+  "theme.common.headingLinkTitle": "קישור ישיר",
+  "theme.common.skipToMainContent": "דלג",
+  "theme.docs.paginator.navAriaLabel": "רשימת דוקומנטאציה",
+  "theme.docs.paginator.next": "הבא",
+  "theme.docs.paginator.previous": "הקודם",
+  "theme.docs.sidebar.collapseButtonAriaLabel": "לסגור",
+  "theme.docs.sidebar.collapseButtonTitle": "לסגור",
+  "theme.docs.sidebar.expandButtonAriaLabel": "לפתוח",
+  "theme.docs.sidebar.expandButtonTitle": "לפתוח",
+  "theme.docs.sidebar.responsiveCloseButtonLabel": "לסגור תפריט",
+  "theme.docs.sidebar.responsiveOpenButtonLabel": "לפתוח תפריט",
+  "theme.docs.versions.latestVersionLinkLabel": "גרסא אחרונה",
+  "theme.docs.versions.latestVersionSuggestionLabel": "לדוקומנטאציה אחרונה, ראה {latestVersionLink} ({versionLabel}).",
```

Comment:
לדוקומנטאציה עדכנית

Reply:
should we remove the placeholders?

Reply:
No sorry, I was on auto mode and forgot the other side isn't Hebrew reading.
It should be, in my opinion:
"לדוקומנטאציה עדכנית, ראה {latestVersionLink} ({versionLabel})."

While the original version does mean "last version" and what I'm suggesting is "up to date version", I think it makes more sense in the context of the language.